### PR TITLE
Create DataCollectionInfoVC

### PR DESCRIPTION
**Pull request by michalsrutek**
- Replaced an info alert with a view controller 
- Needs `dataCollectionMarkdown` added to RemoteConfig @ddmus

### CODE REVIEW

File: BT-Tracking/Interface/Data List/DataCollectionInfoVC.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    // MARK: Setup
+
+    private func initViews() {
+        title = "Informace o sběru dat"
+
+        navigationItem.largeTitleDisplayMode = .never
+        edgesForExtendedLayout = []
+
+        textView.textContainerInset = UIEdgeInsets(top: 30, left: 16, bottom: 16, right: 16)
```
Comment (augard) [inline BT-Tracking/Interface/Data List/DataCollectionInfoVC.swift:34]: You need to set correct right and left margins for iPhone plus size and iPads

File: BT-Tracking/Services/RemoteValues.swift [hunk 2]
```diff
@@ -99,17 +100,26 @@ struct RemoteValues {
...
-        .helpMarkdown: helpMarkdownBackup
+        .helpMarkdown: helpMarkdownBackup,
+        .dataCollectionMarkdown: dataCollectionMarkdownBackup
     ]
 
-    private static var helpMarkdownBackup: String {
+    private static func localValue(forResource resource: String, withExtension extension: String, withKey key: String) -> String {
         guard
-            let path = Bundle.main.url(forResource: "HelpMarkdownBackup", withExtension: "strings", subdirectory: nil),
+            let path = Bundle.main.url(forResource: resource, withExtension: `extension`),
```
Comment (augard) [inline BT-Tracking/Services/RemoteValues.swift:109]: Please use " instand of `

Comment (michalsrutek) [reply BT-Tracking/Services/RemoteValues.swift:109]: then that would be treated as a String, wouldn't it? 🤔  @augard

Comment (michalsrutek) [reply BT-Tracking/Services/RemoteValues.swift:109]: Then that would be treated as a String, wouldn't it? 🤔

Comment (michalsrutek) [reply BT-Tracking/Services/RemoteValues.swift:109]: Then that would be treated as a String, wouldn't it? 🤔 @augard

Comment (augard) [reply BT-Tracking/Services/RemoteValues.swift:109]: Ah, sorry, ignore my comment

File: BT-Tracking/Interface/Data List/DataCollectionInfoVC.swift (hunk 1 above)
Comment (michalsrutek) [reply BT-Tracking/Interface/Data List/DataCollectionInfoVC.swift:34]: Got inspired in `HelpVC`. The constraints and edge insets seem to be the same in there. What would you suggest? @augard

Comment (augard) [reply BT-Tracking/Interface/Data List/DataCollectionInfoVC.swift:34]: I wasn't available when help was merged 😬. So you need to update it from controller.view layoutMargins. And there is override pint in controller "viewLayoutMarginsDidChange" when it's changed.

Comment (augard) [reply BT-Tracking/Interface/Data List/DataCollectionInfoVC.swift:34]: Can you please change it also in helpvc, it should be about 20px for plus phone or iPad, and 16 for other devices.

Comment (michalsrutek) [reply BT-Tracking/Interface/Data List/DataCollectionInfoVC.swift]: @augard Updated. `HelpVC` seems to be doing that by storyboard-ly.

**Pull request merged**